### PR TITLE
Feat: Send A Package entity and auth guard fix

### PR DESCRIPTION
**Pull request by abdul-ahad12**
## Description

SendPackageOrder Entity which aligns with the business needs

### CODE REVIEW

File: apps/backend/backend-main/src/app.module.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ import { UserModule } from "./modules/user/user.module";
     AuthModule,
     AgentModule,
     UserModule,
-    StripeModule,
+    // StripeModule,
```
Comment (shettydev) [inline apps/backend/backend-main/src/app.module.ts:29]: Why was this removed?

File: apps/backend/backend-main/src/entity/Agent.ts
Comment (shettydev) [inline apps/backend/backend-main/src/entity/Agent.ts:1]: If an `Agent` is a type of `User` then shouldn't the user be deleted along with the `Agent`?

File: apps/backend/backend-main/src/entity/AgentDocument.ts
Comment (shettydev) [inline apps/backend/backend-main/src/entity/AgentDocument.ts:1]: Similarly, the Documents related to the Agent should be deleted with `cascade` if the Agent no longer exists

File: apps/backend/backend-main/src/entity/AgentReview.ts
Comment (shettydev) [inline apps/backend/backend-main/src/entity/AgentReview.ts:1]: Same I think the review of the Agent should be deleted, else there will no reference to which user the review belongs to

**Review by shettydev — CHANGES_REQUESTED** [review]

File: apps/backend/backend-main/src/entity/AgentDocument.ts
Comment (novareachsolutions) [reply apps/backend/backend-main/src/entity/AgentDocument.ts:1]: Hey yes, onDelete: "CASCADE", does that only

File: apps/backend/backend-main/src/entity/AgentReview.ts
Comment (novareachsolutions) [reply apps/backend/backend-main/src/entity/AgentReview.ts:1]: Hey yes, onDelete: "CASCADE", does that only

**Review by amitravikumar — APPROVED** [review]

**Review by shettydev — APPROVED** [review]

**Pull request merged**